system-int-test/src/cmr/system_int_test/utils/variable_util.clj

```diff
@@ -158,7 +156,7 @@
   and a user-id."
   [variable concept-id revision-id user-id]
   {:concept-type :variable
-   :native-id (string/lower-case (or (:variable-name variable) ""))
+   :native-id (or (:variable-name variable) "")
```

Comment:
`virable-name` of a variable will never be nil and `native-id` should be the lowercase of `variable-name`, so this line should be `:native-id (string/lower-case (:variable-name variable))`